docs/application/native/guides/ui/efl/wearable/ui-components.md

```diff
@@ -16,6 +16,7 @@ The wearable UI components are designed to allow the user to interact with small
 | Presentation views    | [Win](./component-win.md) | The  win component is the root window component often used in an application. It  allows you to create some content in it, and it is handled by the window  manager. |
 | Presentation  views   | [Background](./component-background.md) | The  background component can be used to set a solid background decoration to a  window or a container object. It works like an image, but has some  background-specific properties, such as setting it to a tiled, centered,  scaled, or stretched mode. |
 | Presentation  views   | [Genlist](./component-genlist.md) | The  genlist component displays a scrollable list of items. It can hold a lot of  items while still being fast and memory-efficient (since only the visible  items are allocated memory). |
+| Presentation views   | [Gengrid](./component-gengrid.md) | The  gengrid component displays objects on a grid layout and renders only the  visible objects. |
```

Comment:
How about following the alphabetical order?

```suggestion
| Presentation views   | [Gengrid](./component-gengrid.md) | The gengrid component displays objects on a grid layout and renders only the visible objects. |
| Presentation views   | [Genlist](./component-genlist.md) | The genlist component displays a scrollable list of items. It can hold a lot of items while still being fast and memory-efficient (since only the visible items are allocated memory). |
```